StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethod.swift

```diff
@@ -210,7 +210,7 @@ public class STPPaymentMethod: NSObject, STPAPIResponseDecodable {
 
     // MARK: - STPAPIResponseDecodable
     /// :nodoc:
-    @objc required init(
+    @objc @_spi(STP) public required init(
```

Comment:
I was trying to figure out why this was needed.  I removed `@_spi(STP) public ` and PaymentSheet Example still compiles -- revert this change?

Reply:
It was for a now-deleted bridge between STPCustomerContext and StripeCustomerAdapter. (STPCustomerContext requires an entire STPPaymentMethod for attach/detach, even though it only uses the ID.) Will delete.